packages/components/ng-ovh-payment-method/src/components/register/sepa-information-modal/template.html

```diff
@@ -0,0 +1,22 @@
+<div data-ng-if="$ctrl.isModalOpened">
+    <div class="modal d-block" role="dialog" id="modal-sepa-information">
+        <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable">
+            <div class="modal-content">
+                <oui-modal
+                    data-heading="{{:: 'ovh_payment_method_register_sepa_information_modal_title' | translate }}"
+                    data-primary-action="$ctrl.onConfirm()"
+                    data-primary-label="{{:: 'ovh_payment_method_register_sepa_information_modal_confirm' | translate }}"
+                    data-on-dismiss="$ctrl.onConfirm()"
```

Comment:
**nitpick:** `onConfirm` is misleading here. Can you rename the name to `closeModal` when user need not confirm.